app-emulation/box64/autobuild/defines

```diff
@@ -0,0 +1,33 @@
+PKGNAME=box64
+PKGSEC=utils
+PKGDES="an userspace x86_64 emulator with a twist, aimed to run x86_64 Linux programs on non-x86_64 Linux systems"
+PKGDEP="glibc gcc-runtime"
+ABTYPE="cmakeninja"
+BUILDDEP="mold"
+
+FAIL_ARCH="(amd64|loongson3)"
+
+# No need to regenerate wrapper,-DCI=ON
+CMAKE_AFTER__PPC64EL=(
```

Comment:
`CMAKE_AFTER__*` is meant as an overlay:

```bash
CMAKE_AFTER=(
    '-DCI=ON'
    '-DWITH_MOLD=ON'
)
CMAKE_AFTER__ARM64=(
    "${CMAKE_AFTER[@]}"
    '-DARM64=ON'
    '-DARM_DYNAREC=ON'
)
...
```

Also, at least sort the variables by name so they are easier to find?